### ISSUE TITLE
`tailwind.light.mailer.config.js` not showing up in gem source

### ISSUE THREAD

**Issue by gazayas** [bullet-train-co/bullet_train-themes-light#22]
This file is in the root directory of the GitHub repository, but for some reason it doesn't show up in the actual source of the gem itself. I stumbled upon this when trying to eject a theme and got this error:
```
cp: cannot stat '/home/gazayas/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/bullet_train-themes-light-1.0.29/tailwind.mailer.light.config.js': No such file or directory
```

However, if you use a local version of the gem in the Gemfile (like `gem "bullet_train-themes-light", path: "./local/bullet_train-themes-light"`) it works without a problem.

**Comment by jagthedrummer** [comment]
I don't think this is relevant anymore. I just tested ejecting a theme from a fresh starter repo project and it worked as expect.
